docs/INSTALL.md

```diff
@@ -0,0 +1,659 @@
+
+<a name="e-mail-configuration-optional"></a>
+## E-Mail Configuration (OPTIONAL)
+
+You MAY use whatever MTA you like to send E-Mail generated by ILIAS. We RECOMMEND to use an already existing smarthost (mailhub). A very simple way to do so is using ```ssmtp```:
+
+On Debian/Ubuntu execute: 
+```
+apt-get install ssmtp
+```
+
+On RHEL/CentOS execute: 
+```
+yum install ssmtp
+```
+
+The configuration file for SSMTP (e.g. ```/etc/ssmtp/ssmtp.conf ```) MAY look as follows:
+
+```
+#
+# Config file for sSMTP sendmail
+#
+# The person who gets all mail for userids < 1000
+# Make this empty to disable rewriting.
+root=yourmail@mail.com
+
+# The place where the mail goes. The actual machine name is required no 
+# MX records are consulted. Commonly mailhosts are named mail.domain.com
+mailhub=smtp.yourmail.com
+
+# Where will the mail seem to come from?
+rewriteDomain=yourservername.org
+
+# The full hostname
+hostname=yourserver.example.com
+
+# Are users allowed to set their own From: address?
+# YES - Allow the user to specify their own From: address
+# NO - Use the system generated From: address
+FromLineOverride=YES
+```
+
+<a name="install-other-depedencies"></a>
+## Install other Depedencies
+
+On Debian/Ubuntu execute: 
+```
+apt-get install zip unzip php5-gd php5-mysql php-xsl imagemagick openjdk-7-jdk
+```
+
+On RHEL/CentOS execute: 
+```
+yum install zip unzip php-gd libxslt ImageMagick java-1.7.0-openjdk
+```
+<a name="installation-wizard"></a>
+## Installation Wizard
+
+After having all dependencies installed and configured you should be able to run the ILIAS Installation Wizard using http://yourservername.org/setup/setup.php
+
+Make sure to reload your Apache configuration before entering the Wizard. Otherwise there are unmet dependencies in the setup (like XLS and GD are both installed but ILIAS does not see them, yet).
+
+<a name="configure-ilias-java-rpc-server-optional"></a>
+## Configure ILIAS Java RPC server (OPTIONAL)
+
+The ILIAS Java RPC server is used for certain OPTIONAL functions as Lucene Search or generating PDF Certificates. To enable the RPC server you need to place a configuration file in ```<YOUR_ILIAS_DIR>/Services/WebServices/RPC/lib/ilServer.properties```:
+
+```
+[Server]
+IpAddress = localhost
+Port = 11111
+IndexPath = /var/www/html/ilias/data/
+LogFile = /var/www/files/ilServer.log
+LogLevel = WARN
+NumThreads = 1
+RamBufferSize = 256
+IndexMaxFileSizeMB = 500
+
+[Client1]
+ClientId = ACMECorp
+NicId = 0
+IliasIniPath = /var/www/html/ilias/ilias.ini.php
+```
+
+ILIAS can generate a proper configuration file via the Administration menu ("Administration -> General Settings -> Server -> Java-Server -> Create Configuration File"). Please note that the configuration file is not directly written to the file system, you MUST copy the displayed content and create the file manually.
+
+You MAY use the following SysV-Initscript to start the RPC server:
```

Comment:
An SystemD configuration file would be nice, since the recommended Linux distributions use SystemD now. Unfortunately I have no such config file at hand, but that should be rather simple (and much more concise).